codecov/settings_base.py

```diff
@@ -304,16 +305,9 @@
     "REDOC_DIST": "SIDECAR",  # serve Redoc from Django (not CDN)
 }
 
```

Comment:
The reason I removed these existing policies is because it wasn't doing anything before this commit. These `CSP_*`  configurations are consumed by the `CSPMiddleware` module, so we probably don't need these as they didn't do anything this whole time.

Reply:
This make sense, I'm wondering if maybe these were meant to be on though at some point but implementation was just never finished 🤔 

Or conversely were never cleaned up

Reply:
Looks like they were added purposefully by Scott ~13 months ago, maybe we should actually keep?

Reply:
Yeah it was purposefully added, but the thing is those CSP headers aren't even in the response because the app doesn't use CSP headers right now. And with this change it would add those CSP headers, so I kind of don't want to introduce all those policies now, not knowing what they would impact.

Reply:
Yeah that's a fair point, we can always add them back too if we want